src/main/index.js

```diff
@@ -1,9 +1,11 @@
 'use strict'
 
 import './globalSetting'
-import { app, Menu } from 'electron'
+import fs from 'fs'
```

Comment:
Pls put native node module `fs` before other modules from npm, but don't put it before `import './globalSetting'`, it maybe cause crash bug.